gapii/cc/spy.cpp

```diff
@@ -186,6 +189,11 @@ Spy::Spy()
                 (header.mFlags & ConnectionHeader::FLAG_DISABLE_PRECOMPILED_SHADERS) != 0;
         mRecordGLErrorState =
                 (header.mFlags & ConnectionHeader::FLAG_RECORD_ERROR_STATE) != 0;
+        // This will be over-written if we also set the header flags
+        mSuspendCaptureFrames = header.mStartFrame;
+        mCaptureFrames = header.mNumFrames;
+        mSuspendCaptureFrames.store((header.mFlags & ConnectionHeader::FLAG_DEFER_START)?
+            -1: mSuspendCaptureFrames.load());
```

Comment:
Maybe make the `-1` a constant defined somewhere with an easy to understand name?